app/assets/javascripts/darkswarm/directives/disable_dynamically.js.coffee

```diff
@@ -0,0 +1,13 @@
+# Allows disabling of link buttons via disabled attribute.
+# This is normally ignored, ie the link appears disabled but is still clickable.
```

Comment:
are you saying "ng-disabled"" doesnt work in these cases? do you know why? It looks like this directive shouldnt be needed.

Reply:
It does not work! It appears to be disabled, but is still clickable. I tried hacking the `href` property instead with a conditional so the link would not function when the conditions are not met, but that didn't work either. :see_no_evil: 

SO discussion here: https://stackoverflow.com/questions/23425254/enable-disable-anchor-tags-using-angularjs